roles/cifmw_cephadm/tasks/rgw_cert.yml

```diff
@@ -0,0 +1,119 @@
+---
+# Copyright 2024 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Check if cifmw_cephadm_certificate path exists
+  ansible.builtin.stat:
+    path: "{{ cifmw_cephadm_certificate }}"
+  register: cert_stat_result
+  when: cifmw_cephadm_certificate | length > 0
+
+- name: Check if cifmw_cephadm_key path exists
+  ansible.builtin.stat:
+    path: "{{ cifmw_cephadm_key }}"
+  register: key_stat_result
+  when: cifmw_cephadm_key | length > 0
```

Comment:
as rgw task comes first, good to have L17 to L27 , but [1] should be removed from 1821
and dashboard cert conditions in [2] and [3] should be updated to use stat.exists

[1] https://github.com/openstack-k8s-operators/ci-framework/pull/1821/files#diff-48e795627a589458ace8f5d0fdbfad1827cc87421722998b61e157e81e9e8ea4R94-R104

[2] https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/cifmw_cephadm/tasks/dashboard/dashboard.yml
[3] https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/cifmw_cephadm/tasks/monitoring.yml#L26

Reply:
I don't know: it depends on the layout: if you do not deploy RGW but you deploy the dashboard you need that kind of check, right?

Reply:
ahh yeah i missed that point, rgw/dashboard deployment can be optional, may be we should move L17 to L27 to pre.yaml  or some other task, so that we can reuse it wherever needed, wdyt ?

Reply:
@fultonj  L17 to L27 can be removed now as [1] has it in pre.yaml.

[1]  https://github.com/openstack-k8s-operators/ci-framework/pull/1821/files#diff-ffe549a4bf8e540dc9d3a37b219441bfaaf6d4e236bb202b6a32d55949e42f56

Reply:
OK, I've removed them.